### ISSUE TITLE
Feature Request: case insensitive match for "shellcheck disable"

### ISSUE THREAD

**Issue by wileyhy** [koalaman/shellcheck#3064]
#### For new checks and feature suggestions
- [X] https://www.shellcheck.net/ (i.e. the latest commit) currently gives no useful warnings about this
- [X] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh

#!/bin/bash

a=( x y z )

# ShellCheck disable=SC2068
echo ${a[@]}

# shellcheck disable=SC2068
echo ${a[@]}

```

#### Here's what shellcheck currently says:

$ shellcheck myscript
 
Line 6:
echo ${a[@]}
     ^-- SC2068 (error): Double quote array expansions to avoid re-splitting elements.

$

#### Here's what I wanted or expected to see:

In "shellcheck disable" statements, the string

    # shellcheck disable=SC####

is recognized, but the string

    # Shellcheck disable=SC####

is not. I expected ShellCheck to recognize capital letters in disable statements.
